src/main/java/inf112/isolasjonsteamet/roborally/players/Player.java

```diff
@@ -34,4 +34,19 @@
 	 * Sets direction of the player.
 	 */
 	void setDir(Orientation dir);
+
+	/**
+	 * Damage a players robot with 1 damage token
+	 */
+	void damageRobot();
+
+	/**
+	 * Repair a players robot with 1 damage token
+	 */
+	void repairRobot();
+
+	/**
+	 * If a player gets 10 damageTokens on his robot, player loose 1 life
```

Comment:
This comment should more describe what the method does (decrements the life, and resets damage tokens)